RawWaterManagement/RegulationStructure/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,111 @@
+        -47.6079975,
+        -48.871772
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Nature big watch better.",
+      "addressLocality": "Marriage enjoy line without crime. Fund herself suddenly there. Four money represent argue hear million. Full spend stand its security month.",
+      "addressRegion": "Race religious believe hotel explain make. Specific design modern great. Audience study generation seek center color there.",
+      "addressCountry": "Provide officer call carry she middle tough. Although common beyond day protect without. Course indeed specific.",
+      "postalCode": "Gun fight speech tough process air process. Outside in attention with.",
+      "postOfficeBoxNumber": "Nation young media address interest course every. Small however peace price anyone. Talk bank pretty.",
+      "areaServed": "Same job company and effect."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Cell seek area drop media. If spring story address share dream however mouth."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:MMYA:05439087",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1971-06-03T12:15:35Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1984-11-25T18:10:41Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Peace third agent approach carry company become own. Own cultural writer in floor method see."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Treatment notice foot finish source. Win least tell art voice article summer."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "South ten hundred rock information knowledge art. Environment evening relationship."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Drop performance purpose mother there various conference. Product increase notice policy couple action here somebody. Country field environment learn."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Hundred form trade involve. Will law still reduce knowledge. Soon major treat."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:HAWW:05049374",
+      "urn:ngsi-ld:dataModel:items:UMXA:22113309"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:XGXY:01008206",
+      "urn:ngsi-ld:dataModel:items:XIGV:76659038"
+    ]
+  },
+  "type": "RegulationStructure",
+  "tag": {
+    "type": "Property",
+    "value": "Since college or wear TV respond himself. Pattern area drive year upon."
+  },
+  "numberOfGates": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "numberOfSpillways": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "teleCommand": {
+    "type": "Property",
+    "value": [
+      "false"
+    ]
+  },
+  "hasSluiceGate": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}